packages/picker/test/picker.test.ts

```diff
@@ -679,7 +693,7 @@ describe('Picker', () => {
         const el = await pickerFixture();
         el.addEventListener('change', (event: Event) => {
             const { value } = event.target as Picker;
-            console.log('change', value);
+            //console.log('change', value);
```

Comment:
these seemed noisy, but probably worth leaving in the code since it's useful for debugging. Happy to re-enable though!